config/admin_ranks.txt

```diff
@@ -43,15 +45,13 @@ Game Master		= +EVERYTHING *EVERYTHING
 Lazy Master 	= +EVERYTHING -AUTOLOGIN *EVERYTHING
 Three Dog		= -@ -ADMIN +DJ
 
-Head Coder      = +FUN +STEALTH +POSSESS +REJUV +BUILD +SERVER +DEBUG +VAREDIT +SOUND +SPAWN +RIGHTS +DJ +POLL
-Coder			= +DEBUG +VAREDIT +SERVER +SPAWN +POLL -AUTOLOGIN
+Scribe 			= +SERVER +DEBUG +VAREDIT +SPAWN -AUTOLOGIN
+Head Scribe 	= +@ +POLL -AUTOLOGIN
```

Comment:
This removes the additional abilities of Scribe and Head Scribe routinely used for testing.

Reply:
> This removes the additional abilities of Scribe and Head Scribe routinely used for testing.

> +DEBUG +VAREDIT +SERVER +SPAWN +POLL -AUTOLOGIN (this would be the code before Johns changes which me and Ren have gone though and refined)

This would make more sense if this was the case of the old perms. most of these new perms were added only recently through Johns additions, you all did just fine with the previous perms supplied to you before this, as such it has been updated so the coders can do active testing on server, anything else you would like can be done through a local host as the tools supplied for the live server are enough.

Reply:
Head Scribes are generally trusted deployers who can safely be afforded +EVERYTHING. If we don't believe someone can effectively handle that permission level, we shouldn't make them Head Scribe.

I would still give regular Scribe polling power, they may wish to ask about the server's willingness to have a reset for updates.

Reply:
The Head Coder change came from me and was a dupe/rename of Game Master to clarify folks on the dev side. It wasn't part of John's changes and shouldn't be reverted.

Reply:
Personal preference for you on trusting Head scribes with anything more than what they need, they have been given the tools they need and that is enough, my apologies but it is not for the coders to do anything other than **what they signed up to do**.

Reply:
It's certainly within the realm of what we need and routinely use. I've answered player questions with +ADMIN, bulk spawned for testing with +BUILD, fixed heavily varedited damaged things with +REJUV. I have an entire directory of Fallout .ogg files for +DJ. There's a whole category of debug actions that are inexplicably in +FUN (due to being in Secrets).

Reply:
Half of this stuff you explained can be done and should be done in a localhost.
+DJ is more a tool for fun than a debug tool.

Reply:
As I have stated before, you got just by fine without issues with the previous tools at your disposal.
I mean not to push or prod but this is only stalling on something myself and Ren have unanimously agreed on.

Reply:
Just because we're doing local testing doesn't mean there aren't benefits to public testing in a populated server with players. Also, fun is why we're here.

Reply:
Actually, I talked with Ren and he asked me to get this discussion rolling, because coders are going to know a lot better what they regularly use. The previous tools at my disposal were +EVERYTHING, so this is absolutely a reduction in what I can do for the server.

Reply:
hey look i can do value changes im a coder, coder rank pls thx THANKS FOR FREE FLAGS!! hashtag coderbus.

Reply:
> Now let's put this in the correct place.
> 
> After all roles are moved over and the transition period is smoothed out, all old roles are supposed to be deleted unless someone wishes to keep them for historical reasons.
> 
> The exceptions are Three Dog (it's already fallout), Host (I believe it plugs into other stuff so removing it would cause problems) and possibly admin (for the same reason).
> 
> I referenced this in the previous PR when warning John not to delete roles people were currently configured to because it would lead to headaches and problems.

>Admin Observer	= -AUTOLOGIN
Moderator		= +ADMIN
Admin Candidate	= +@
Trial Admin		= +@ +SPAWN +VAREDIT +BAN
Badmin			= +@ +POSSESS +POLL +BUILDMODE +SERVER +FUN
Admin			= +@ +STEALTH +SOUNDS +DEBUG
Game Master		= +EVERYTHING *EVERYTHING
Lazy Master 	= +EVERYTHING -AUTOLOGIN *EVERYTHING
Three Dog		= -@ -ADMIN +DJ

these are to be cleared up to reduce bloat, Host is at the bottom under Elder and Threedog isnt linked to anything the Perms to it are which is +DJ and can be applied to any role.

Reply:
Yes fun is why your here, though the fun should be in playing the server not any badmining, the tools supplied to you for the live server are enough for anything that needs to be done outside of localhosting.

Reply:
All right, I can see your perspective. We'll give @RenGusta some time to read and decide. Happy to approve if I get his confirmation that this is the stance he agrees with. If not, proposed change of Head Scribe back to +EVERYTHING (or +EVERYTHING minus rights).

Reply:
So we done stalling here folks?

Reply:
There is no reason for general coders to have +FUN or any of the relating "fun" admin tools. If you want to dick with the players become more than just a coder. Having all of those fluff roles just boggs things down and creates a level of confusion in regards to who should/can do what. 

Just looking at some of the complaints (one below), they seem to be admin specific duties anyways, somewhat proving my point. If you find yourself in the rare situation where you honestly need an admin function on the live production environment, I don't see why you wouldn't be able to just ask for an admin to help you test something.
>  I've answered player questions with +ADMIN

Reply:
Time to let it go, the majority have agreed with the changes along with people from other places. If you disagree with the changes, time to go elsewhere or bring it up another time.